### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by pitr-ch**

### CODE REVIEW

**Comment by pitr-ch** [comment]
@iNecas finally updated

File: lib/dynflow/world.rb [hunk 1]
```diff
@@ -52,7 +52,14 @@ def subscribed_actions(action_class)
 
     # reload actions classes, intended only for devel
     def reload!
-      @action_classes.map! { |klass| klass.to_s.constantize }
+      # TODO what happens with newly loaded classes
+      @action_classes.map! do |klass|
```
Comment (iNecas) [inline lib/dynflow/world.rb]: Should not we remove the missing classes from the array, to not rescue from the load error every time?

**Comment by iNecas** [comment]
One minor comment

**Comment by pitr-ch** [comment]
replaced by https://github.com/iNecas/dynflow/pull/1

**Pull request closed without merging**